src/main/java/chess/domain/piece/pawn/Pawn.java

```diff
@@ -0,0 +1,28 @@
+package chess.domain.piece.pawn;
+
+import chess.domain.color.Color;
+import chess.domain.piece.Direction;
+import chess.domain.piece.Piece;
+import chess.domain.piece.Position;
+import java.util.Set;
+
+public abstract class Pawn extends Piece {
+    private final Set<Direction> directions;
+
+    protected Pawn(Position position, Color color, Set<Direction> directions) {
+        super(position, color);
+        this.directions = directions;
+    }
+
+    @Override
+    public Set<Position> findPathTo(Position destination) {
+        Set<Position> movable = position.findMovablePositions(directions);
+
+        if (!movable.contains(destination)) {
+            throw new IllegalArgumentException("이동할 수 없습니다.");
+        }
+        return position.findCourses(position.findDirectionTo(destination), destination);
```

Comment:
```suggestion
        return position.findCourses(destination);
```
position 에게 행위를 위임하는김에 `findDirectionTo` 행위까지 위임해보죠!

Reply:
앗 인자에 들어갈 필요가 없었네요..!! 이부분도 반영하겠습니다!